channeldb/kvdb/etcd/stm.go

```diff
@@ -419,6 +430,11 @@ func (s *stm) Get(key string) ([]byte, error) {
 
 	if len(kvs) > 0 {
 		return []byte(kvs[0].val), nil
+	} else {
+		// Add assertion to the read set.
```

Comment:
what is this asserting exactly?

Reply:
We add the key to the read set with 0 mod revision in order to create a constraint upon commit that the key does not exits. Extended the comment above, PTAL.